### ISSUE TITLE
apidoc: "Last updated 2030-01-01 00:00:00"

### ISSUE THREAD

**Issue by blueyed** [awesomeWM/awesome#2108]
The apidocs (e.g. https://awesomewm.org/apidoc/classes/awful.wibar.html) have "Last updated 2030-01-01 00:00:00" in the footer now.

Seems to come via 5ac6db569.

**Comment by psychon** [comment]
Wasn't that (almost) the whole point of the commit? Since this is only the api docs for the in-devel version, I see no problem with that. Plus, it allows us to simplify the script that pushes it into git since all the magic that tries to filter out these date changes is no longer needed.

**Comment by blueyed** [comment]
Then I would rather remove it from the template.
It was meant to fix the example in the doc only.

**Comment by psychon** [comment]
Something like the following, perhaps?
```diff
diff --git a/.travis.yml b/.travis.yml
index 84609943..30b402f4 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -173,6 +173,10 @@ script:
       cd "$BUILD_IN_DIR"
       travis_run_in_fold "build_in_dir" cmake $CMAKE_ARGS "$SOURCE_DIRECTORY"
     fi
+  - |
+    if [ "$BUILD_APIDOC" = "true" ]; then
+      SOURCE_DATE_EPOCH= make ldoc
+    fi
   - travis_run_in_fold "make" make
   - |
     if [ "$TRAVIS_TEST_RESULT" = 0 ]; then
```
(Or if that doesn't work: `(unset SOURCE_DATE_EPOCH && make ldoc)`)

**Comment by blueyed** [comment]
Yeah, `env -u SOURCE_DATE_EPOCH make ldoc` could also be used then possibly.
